src/lib/transition_frontier/extensions/ledger_table.ml

```diff
@@ -18,6 +18,8 @@ module T = struct
 
   type view = unit
 
+  let name = "ledger_table"
```

Comment:
Thinking out loud: could we automagically derive the name from ``__FILE__`` as default behavior? At best it's a follow-up issue.